src/types.ts

```diff
@@ -188,6 +188,9 @@ export interface PostHogConfig {
     advanced_disable_toolbar_metrics: boolean
     feature_flag_request_timeout_ms: number
     get_device_id: (uuid: string) => string
+    segment_config?: {
+        user_id?: string
+    }
```

Comment:
how does this value get set?  Like I see how we're reading it in`segment-integration`, but if we're adding this new payload to the `PostHogConfig`, how does the value appear here in the first place?

Reply:
This should be clarified in 2ec3981